### PR TITLE
Fixed popper dot command

### PR DESCRIPTION
**Pull request by JayjeetAtGithub**
No more prompts for `secrets` during `popper dot`.

### CODE REVIEW

**Comment by JayjeetAtGithub** [comment]
@ivotron Can't understand why the build failed in [this](https://travis-ci.org/systemslab/popper/jobs/524910733) case ?

File: ci/test/dot [hunk 1]
```diff
@@ -9,7 +9,7 @@ cd "$test_repo_path"
 git clone https://github.com/cplee/github-actions-demo.git
 cd github-actions-demo
 
-export PHONY_SECRET=foo
+# export PHONY_SECRET=foo
```
Comment (ivotron) [inline ci/test/dot]: we can delete these lines, as `popper dot` won't prompt for them anymore

File: cli/popper/gha.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ class Workflow(object):
     """
 
     def __init__(self, wfile, workspace, dry_run,
-                 reuse, parallel):
+                 reuse, parallel, no_prompt=False):
```
Comment (ivotron) [inline cli/popper/gha.py]: maybe rename to `no_secrets_prompt` or `skip_secrets_prompt`?

**Comment by ivotron** [comment]
> Can't understand why the build failed in [this](https://travis-ci.org/systemslab/popper/jobs/524910733) case ?

not sure either... hmm, maybe a transient issue?

**Comment by JayjeetAtGithub** [comment]
> > Can't understand why the build failed in [this](https://travis-ci.org/systemslab/popper/jobs/524910733) case ?
> 
> not sure either... hmm, maybe a transient issue?

Ya maybe

**Comment by JayjeetAtGithub** [comment]
@ivotron Somehow the arrangement of imports again got disturbed, so fixed that.

**Pull request merged**